testsuite/gnat2goto/tests/separate_subprog/test.out

```diff
@@ -0,0 +1,3 @@
+[2] file p-inc.asu line 6 assertion: SUCCESS
+[1] file p.adb line 14 assertion: FAILURE
```

Comment:
Why do we want this to fail?

Reply:
We do not want it to fail.  It should succeed if the possibility of an overflow is ignored.  I have split the test into 2.  One with tis result so that we can test for regressions and one with the correct result with I have marked with an XFAIL.

Reply:
Thanks.